otter_welcome_buddy/graphql/schema/leetcode.gql

```diff
@@ -0,0 +1,357 @@
+scalar Timestamp
+
+enum DifficultyOption {
+  All
+  Easy
+  Medium
+  Hard
+}
+
+type QuestionCountByDifficultyNode {
+  difficulty: DifficultyOption!
+  count: Int!
+}
+
+type SubmissionCountNode {
+  difficulty: String!
+  count: Int!
+  submissions: Int!
+}
+
+type UserSubmitStatsNode {
+  totalSubmissionNum: [SubmissionCountNode!]!
+  acSubmissionNum: [SubmissionCountNode!]!
+}
+
+type UserContributionNode {
+  points: Int!
+}
+
+type UserBadgeNode {
+  name: String!
+  expired: Boolean!
+  hoverText: String!
+  displayName: String!
+  icon: String!
+}
+
+type TagProblemsCountNode {
+  tagName: String!
+  tagSlug: String!
+  problemsSolved: Int!
+}
+
+type TagProblemCountsCategoryNode {
+  advanced: [TagProblemsCountNode!]!
+  intermediate: [TagProblemsCountNode!]!
+  fundamental: [TagProblemsCountNode!]!
+}
+
+type LanguageProblemCountNode {
+  languageName: String!
+  problemsSolved: Int
+}
+
+type UserProfileNode {
+  ranking: Int!
+  userAvatar: String!
+  realName: String!
+  aboutMe: String!
+  school: String
+  websites: [String!]!
+  countryName: String
+  company: String
+  jobTitle: String
+  skillTags: [String!]!
+  postViewCount: Int!
+  postViewCountDiff: Int!
+  reputation: Int!
+  reputationDiff: Int!
+  solutionCount: Int!
+  solutionCountDiff: Int!
+  categoryDiscussCount: Int!
+  categoryDiscussCountDiff: Int!
+}
+
+type UserNode {
+  username: String!
+  profile: UserProfileNode!
+  contributions: UserContributionNode!
+  githubUrl: String
+  twitterUrl: String
+  linkedinUrl: String
+  # Encode JSON object where each entry represent:
+  # {
+  #   timestamp: # submissions
+  # }
+  submissionCalendar: String!
+  submitStatsGlobal: UserSubmitStatsNode
+  submitStats: UserSubmitStatsNode
+  activeBadge: UserBadgeNode
+  contestBadge: UserBadgeNode
+  tagProblemCounts: TagProblemCountsCategoryNode!
+  languageProblemCount: [LanguageProblemCountNode!]!
+  isDiscussAdmin: Boolean!
+  isDiscussStaff: Boolean!
+  isActive: Boolean!
+  # nameColor - nullable
```

Comment:
Any reason why this line was commented but no explanation of why we should keep it in the codebase?

Reply:
I keep it for reference, is a field but not sure why is referring to